index.html

```diff
@@ -10,17 +10,54 @@
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
 	</head>
 	<body>
 		<div id="container">
```

Comment:
Aaron,
I've made some changes to the code and hit sync in the github desktop, not
sure if that made it commit the changes in github.

Best Regards,
Andrew Filippi

On Mon, Mar 7, 2016 at 5:33 PM, Aaron Cordova <notifications@github.com>
wrote:

> In index.html
> <https://github.com/FEWD80/homework-02-relaxr-landing-page/pull/8#discussion_r55285125>
> :
>
> > @@ -10,17 +10,54 @@
> >  		<!-- external CSS link -->
> >  		<link rel="stylesheet" href="css/reset.css">
> >  		<link rel="stylesheet" href="css/style.css">
> > -
> > +		<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
> >  	</head>
> >  	<body>
> >  		<div id="container">
>
> @filippia <https://github.com/filippia> The good thing is you're using
> this div to house the rest of the content on the page. You should further
> use it to you're advantage by putting the 960px width on it. Remember, when
> we look at a web page, we see boxes everywhere. Break down you're page into
> manageable "boxes" and work from there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEWD80/homework-02-relaxr-landing-page/pull/8/files#r55285125>
> .
>